homeassistant/components/sensor/neurio_energy.py

```diff
@@ -21,48 +24,133 @@
 CONF_API_SECRET = 'api_secret'
 CONF_SENSOR_ID = 'sensor_id'
 
-DEFAULT_NAME = 'Energy Usage'
+ACTIVE_NAME = 'Energy Usage'
+DAILY_NAME = 'Daily Usage'
```

Comment:
I assume you mean "Energy Usage" and "Daily Energy Usage" as the names?  Works for me. 

By the way, just realized you are the original author for this component, thanks for putting in the effort to add support for Neurio in the first place.